locust/static/locust.js

```diff
@@ -227,10 +227,19 @@ $('#import_csv_btn').click(function(event) {
 
 $('#multiple_column_form').submit(function(event) {
     event.preventDefault();
+    $("#multiple_hidden_config_json").val(JSON.stringify(json_editor.get(), null , 4));
     $.post($(this).attr("action"), $(this).serialize(),
         function(response) {
             if (response.success) {
-                location.reload(true);
+                try{
+                    json_editor.set(JSON.parse(response.data));
+                    $(".multiple_column").hide();
+                    $("#column_name").empty();
+                    document.getElementById("import_csv_form").reset();
+                }
+                catch(err){
+                    alert(err.message);
+                }
```

Comment:
when no csv and json keys is present, the message should show that it is required fields, not Convert error : ''